showkase/src/main/java/com/airbnb/android/showkase/ui/ShowkaseBrowserApp.kt

```diff
@@ -186,19 +189,34 @@ private fun ShowkaseAppBarTitle(
     }
 }
 
+val LineCountKey = SemanticsPropertyKey<Int>("lineCount")
+var SemanticsPropertyReceiver.lineCountVal by LineCountKey
+
 @Composable
 fun ToolbarTitle(
     string: String,
     modifier: Modifier
 ) {
+    val lineCount = remember {
```

Comment:
Not ideal to have test logic live in the component directly but I guess that happens pretty often in Compose due to how semantics are setup with some hooks into testing. Don't have a better idea on testing this so might be fine for now.

Reply:
Totally agree with you on that! I think this can work for now :)